0.2.4-dev compile error: -Werror=cast-function-type

Currently getting this compile error on 0.2.3 and 0.2.4-dev

```
[ 54%] Building CXX object src/radiotray-ng/extras/media_keys/CMakeFiles/media_keys.dir/linux/media_keys.cpp.o
In file included from /usr/include/sigc++-2.0/sigc++/sigc++.h:104,
                 from /usr/include/glibmm-2.4/glibmm/thread.h:49,
                 from /usr/include/glibmm-2.4/glibmm.h:88,
                 from /usr/include/giomm-2.4/giomm.h:22,
                 from /home/build/YPKG/root/radiotray-ng/build/radiotray-ng.git/src/radiotray-ng/extras/media_keys/linux/media_keys.cpp:23:
/usr/include/sigc++-2.0/sigc++/signal.h: In static member function ‘static sigc::internal::signal_emit0<void, sigc::nil>::result_type sigc::internal::signal_emit0<void, sigc::nil>::emit(sigc::internal::signal_impl*)’:
/usr/include/sigc++-2.0/sigc++/signal.h:794:56: error: cast between incompatible function types from ‘sigc::internal::hook’ {aka ‘void* (*)(void*)’} to ‘sigc::internal::signal_emit0<void, sigc::nil>::call_type’ {aka ‘void (*)(sigc::internal::slot_rep*)’} [-Werror=cast-function-type]
           (reinterpret_cast<call_type>(slot.rep_->call_))(slot.rep_);
                                                        ^
/usr/include/sigc++-2.0/sigc++/signal.h: In static member function ‘static sigc::internal::signal_emit0<void, sigc::nil>::result_type sigc::internal::signal_emit0<void, sigc::nil>::emit_reverse(sigc::internal::signal_impl*)’:
/usr/include/sigc++-2.0/sigc++/signal.h:821:55: error: cast between incompatible function types from ‘sigc::internal::hook’ {aka ‘void* (*)(void*)’} to ‘sigc::internal::signal_emit0<void, sigc::nil>::call_type’ {aka ‘void (*)(sigc::internal::slot_rep*)’} [-Werror=cast-function-type]
           (reinterpret_cast<call_type>(it->rep_->call_))(it->rep_);
                                                       ^
cc1plus: all warnings being treated as errors
make[2]: *** [src/radiotray-ng/extras/media_keys/CMakeFiles/media_keys.dir/build.make:63: src/radiotray-ng/extras/media_keys/CMakeFiles/media_keys.dir/linux/media_keys.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:540: src/radiotray-ng/extras/media_keys/CMakeFiles/media_keys.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
```

Removing `-Werror` in CMakeLists.txt will fix the compile error, but of course that isn't a proper fix.

Thanks in advance!

gcc version: 8.2.0
glibmm : 2.56.0
libsigc++ : 2.10.0

--- a/src/radiotray-ng/CMakeLists.txt
+++ b/src/radiotray-ng/CMakeLists.txt
@@ -37,4 +37,4 @@ target_link_libraries(
 )
 
 add_dependencies(radiotray-ng user-agent)
-target_include_directories(radiotray-ng PRIVATE ${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${GSTREAMER_INCLUDE_DIRS} ${APPINDICATOR_INCLUDE_DIRS} ${GIOMM_INCLUDE_DIRS} ${GLIBMM_INCLUDE_DIRS})
+target_include_directories(radiotray-ng SYSTEM PRIVATE ${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${GSTREAMER_INCLUDE_DIRS} ${APPINDICATOR_INCLUDE_DIRS} ${GIOMM_INCLUDE_DIRS} ${GLIBMM_INCLUDE_DIRS})

--- a/src/radiotray-ng/extras/media_keys/CMakeLists.txt
+++ b/src/radiotray-ng/extras/media_keys/CMakeLists.txt
@@ -8,4 +8,4 @@ add_library(media_keys STATIC ${SRC_FILES})
 
 # todo: if appindicator add the includes
 target_link_libraries(media_keys ${APPINDICATOR_GUI} ${GIOMM_LIBRARIES})
-target_include_directories(media_keys PRIVATE ${GIOMM_INCLUDE_DIRS} ${APPINDICATOR_INCLUDE_DIRS} )
+target_include_directories(media_keys SYSTEM PRIVATE ${GIOMM_INCLUDE_DIRS} ${APPINDICATOR_INCLUDE_DIRS} )

--- a/src/radiotray-ng/extras/rtng_dbus/CMakeLists.txt
+++ b/src/radiotray-ng/extras/rtng_dbus/CMakeLists.txt
@@ -3,4 +3,4 @@ FILE(GLOB SRC_FILES "*.cpp")
 add_library(rtng_dbus STATIC ${SRC_FILES})
 
 target_link_libraries(rtng_dbus ${GLIBMM_LIBRARIES} ${GIOMM_LIBRARIES})
-target_include_directories(rtng_dbus PRIVATE ${GIOMM_INCLUDE_DIRS} ${GLIBMM_INCLUDE_DIRS})
+target_include_directories(rtng_dbus SYSTEM PRIVATE ${GIOMM_INCLUDE_DIRS} ${GLIBMM_INCLUDE_DIRS})

